### ISSUE TITLE
[Benefit Detail Pages] [FE] Search results count is difficult to understand using screen reader

### ISSUE THREAD

**Issue by klwatanabe** [department-of-veterans-affairs/va.gov-cms#23868]
## Current Behavior
The search results count is difficult to understand when using Mac's screen reader functionality. 

**Note:** This issue exists in Content Build and Next Build.

## Expected Behavior
User should be able to understand how many search results have been returned, as read by the screen reader.  

## Reproduction Steps
1. Navigate to https://www.va.gov/forms/
2. Select the search bar to search for VA Forms
3. Enable screen reader functionality on a mac(command + F5)
4. Type "disability" into the search bar and hit return
5. Label displays 'Showing 1 – 10 of 12 results for **"disability"**'
6. Screen reader reads this as 'heading level 2 Showing, level 11, level 1 - , level 1 10, level 1 space, level 1 of, level 112, level 1 results for", level 1 disability, level 1", level 1, 10 items, article'

<img width="689" height="202" alt="Image" src="https://github.com/user-attachments/assets/5572ea81-177c-4145-95d0-9a7b13a1a88a" />

<img width="1067" height="859" alt="Image" src="https://github.com/user-attachments/assets/f654b0d2-616a-4713-a537-e6af9e591f94" />

<img width="762" height="246" alt="Image" src="https://github.com/user-attachments/assets/3dd8ffe1-5e6b-42ee-ab05-c3ae53435894" />
